prow/spyglass/artifacts.go

```diff
@@ -57,14 +57,24 @@ func (s *Spyglass) ListArtifacts(src string) ([]string, error) {
 	return artifactNames, nil
 }
 
+// KeyToJob takes a spyglass URL and returns the jobName and buildID.
+func (*Spyglass) KeyToJob(src string) (jobName string, buildID string, err error) {
```

Comment:
Can we get some tests for input and output here?

Reply:
We have unit tests now.